target/linux/ramips/mt7620/base-files/etc/uci-defaults/03_wireless

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# The TrendNet THA103AC has no ethernet port and to connect to it we need WiFi enabled and for clarity let's seperate 2 and 5 GHZ
+
+. /lib/functions.sh
+
+case "$(board_name)" in
+	trendnet,tha103ac)
+		uci set wireless.radio0.disabled='0'
+		uci set wireless.default_radio0.ssid='OpenWrt5GHZ'
+		uci set wireless.radio1.disabled='0'
+		uci set wireless.default_radio1.ssid='OpenWrt2GHZ'
+		uci commit wireless
+		;;
+esac
```

Comment:
I do not like that the wireless interface is activated by default, but this seams to be the only working interface after OpenWrt installation. 

How do we handle this on other devices which only support wifi?

Reply:
> How do we handle this on other devices which only support wifi?

It looks like we haven't had such devices officially supported yet. Just an idea: reassign WPS button to WiFi enabled/disable button. What do you think?

Reply:
yes that would be a good idea. 

Please also do not change the ssid.

Reply:
@diepeterpan how do you like this option? Changes `03_wireless` can be dropped if you reassign the wps button function.

Reply:
Should be done, all good.